.pre-commit-config.yaml

```diff
@@ -30,11 +30,16 @@ repos:
         pass_filenames: true
         types: [file, javascript]
         language: system
-    -   id: commit-lint
-        name: Commit Lint
-        description: Runs commitlint against the commit message.
+    -   id: conventional
+        name: Conventional Commits
+        description: Ensures commits are conventional.
         language: system
-        entry: bash -c "npm install @commitlint/config-conventional @commitlint/cli; cat $1 | npx commitlint"
-        args: [$1]
-        stages: [commit-msg]
+        pass_filenames: false
+        entry: nix-shell default.nix -A convco --run "convco check || true"
```

Comment:
I have this bypassed right now, as many old commits don't pass conventionality, do we have a plan to go groom history or resolve this somehow?

Reply:
maybe bring this up on the backend channel?

Reply:
@gila do you have thoughts on how to handle commits with invalid conventional commit formatting?